front/src/components/boxs/device-in-room/DevicesBox.jsx

```diff
@@ -92,13 +144,30 @@ class DevicesComponent extends Component {
 
   //Remove x, y when DeviceInRoom is rewrite without action
```

Comment:
is it possible now to remove x, y arguments ?

Reply:
yes, done!